### PR TITLE
TEA GAME CLASSIC RULES

### PR DESCRIPTION
**Pull request by KenanYusuf**
THIS GAME HAS SPIRALLED OUT OF CONTROL AND IT IS ABOUT TIME WE BROUGHT IT BACK TO THE GOOD OLD DAYS.

### CODE REVIEW

**Comment by dompainter** [comment]
👎  nah

**Comment by KenanYusuf** [comment]
👍

**Comment by KenanYusuf** [comment]
👍

**Comment by KenanYusuf** [comment]
👍

**Comment by Asheboy** [comment]
:-1: Sorry, but I like the birthday rule. I just dislike Dom.

**Comment by davidhigginson** [comment]
👎 I'm tired of people purging things

**Comment by jack828** [comment]
👎  I also like the birthday rule

**Comment by KenanYusuf** [comment]
The revolution is coming.

File: README.md [hunk 1]
```diff
@@ -72,18 +53,12 @@ Valid orders include:
...
   Other types of drink (nespresso, hot chocolate, etc.) can be made at the loser’s discretion and are optional.
 
-## Guest Players
-Players not in the usual crowd (above) can play as along as agreed by the above, and provided they abide by the rules.
-
 ## Incorrect Tea Orders
 If an incorrect tea order has been delivered it must be replaced by the correct tea order as soon as the error is reported to the tea maker. It is the responsibility of the tea maker to keep track of all tea orders.
 
 ## Tea Delivery Time
 If a player knows that they can't start making the tea within 10 minutes of the game ending, then they should announce this before play begins. This gives other players the opportunity to withdraw from the game if wanted. If a player does not announce this, and it is evident that they knew of their time delay before the game began, then the player forfeits their exemption.
```
Comment (richhastings) [inline README.md]: I would remove this section

File: README.md [hunk 2]
```diff
@@ -72,18 +53,12 @@ Valid orders include:
...
 
-## Happy Birthday 🎂
-Players are exempt on their birthday and Clock birthday (the day they joined the tea game). 
-
 ## 'Bring out your dead' Rule
```
Comment (richhastings) [inline README.md]: This can be removed too, as it's covered in the `let's play` section.

**Pull request closed without merging**